src/scripts/main.js

```diff
@@ -1,7 +1,212 @@
 'use strict';
+/* eslint-disable */
+const refs = {
+  startBtn: document.querySelector('.start'),
+  fieldCells: document.querySelectorAll('.field-cell'),
+  fieldRows: document.querySelectorAll('.field-row'),
+  messageStart: document.querySelector('.message-start'),
+  messageLose: document.querySelector('.message-lose'),
+  messageWin: document.querySelector('.message-win'),
+  score: document.querySelector('.game-score'),
+};
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const WIN_VALUE = 2048;
+const FIELD_LENGTH = 15;
```

Comment:
The FIELD_LENGTH constant is set to 15, but the typical 2048 game uses a 4x4 grid. Ensure that this constant matches the intended size of your game field.